library/core/src/ptr/mod.rs

```diff
@@ -893,8 +895,9 @@ pub unsafe fn write<T>(dst: *mut T, src: T) {
     // `dst` cannot overlap `src` because the caller has mutable access
     // to `dst` while `src` is owned by this function.
     unsafe {
-        copy_nonoverlapping(&src as *const T, dst, 1);
-        intrinsics::forget(src);
+        // FIXME: No need for mutable borrow once #80418 hits beta
+        copy_nonoverlapping(&mut src as *const T, dst, 1);
```

Comment:
When should this happen? :) Or where can I look to find out?

Reply:
https://forge.rust-lang.org/ shows when the next no-breakage-week is, so I guess February 9th?

Reply:
Since we are getting closer to the 9th, would it make sense to undo the `&src` -> `&mut` changes, change the PR title and perhaps this could be seen as an actual (non-experiment) PR? :)

Reply:
well, it makes little sense to do the change before the bootstrap compiler is updated, otherwise you'll be battling differing compiler versions and that's just a mess. I'm not quite sure how long after the 9th it takes for that to happen, but it shouldn't be too long.